citi-onboarding/DADe-onepage
# Feature/banner update and mobile version

## What I did

- Update banner to look like the new mockup
- Add mobile version of banner
- Adapt banner to stay on the right side of the screen

| Mockup | Real |
|--------|------|
|![Captura de Tela_select-area_20210622142216](https://user-images.githubusercontent.com/61971582/122977343-a8afff80-d36b-11eb-8896-5d4d8f68b926.png)|![Captura de Tela_select-area_20210622153042](https://user-images.githubusercontent.com/61971582/122980299-dba7c280-d36e-11eb-8174-311797bb81b6.png)|

|

| Mockup | Real |
|--------|------|
|![Captura de Tela_select-area_20210622142202](https://user-images.githubusercontent.com/61971582/122977407-c0878380-d36b-11eb-88d0-f9db24698a7d.png)|![Captura de Tela_select-area_20210622153022](https://user-images.githubusercontent.com/61971582/122980387-f1b58300-d36e-11eb-8b46-88aeb3ffbaa1.png)|


## How to test

1. Open the terminal and go to the folder of this project

1. In terminal:
   1. `git checkout develop`
   1. `git pull`
   1. `git checkout feature/banner-update-and-mobile-version`
   1. `git pull origin feature/banner-update-and-mobile-version`
   1. `npm install`
   1. `cd server/`
   1. `node index.js`
   2. Go to a new terminal
   3. `cd client/`
   4. `npm start`

1. In your browser go to http://localhost:3000 (should already be open by the `npm start` command) 

1. Check if the component looks just like the mockup in different screen sizes, and if the right side of the banner stays at the right side of the screen in widths under 1440px (aproximation). 
-obs: Some sections are too big, I suggest holding control + scrolling the mouse to get some zoom in, and click "Redefine" in the browser zoom selection, to get a better look. Scrolling horizontally might help.

1. Read the changed files section on GitHub 

1. Test the things that you think are worth testing, even the ones that are not in this description
